frontend/src/static/app/api/api-types.ts

```diff
@@ -13,3 +13,9 @@ type IntroClassificationsResult = {
   targetLanguageMap: { [key: string]: string }
   targetContainerMap: { [key: string]: string }
 }
+
+type LogBean = {
+  fileName: string
+  content: string
+}
+type LogListResult = Array<LogBean>
```

Comment:
配列の型定義は以下の方がよく使うかもです！
```suggestion
type LogListResult = LogBean[]
```